### DISCUSSION TITLE
Does a Reverted Pull Request Still Count Towards My GSSoC Leaderboard Score?

### DISCUSSION THREAD

**Discussion by dhruv389** [Note-Vault/Note-Vault#284] [Q&A] [answered]
If my pull request gets merged but is later reverted due to an issue by project admins , will it still be counted towards my GSSoC leaderboard score?
Please give the answer of my question.
@VaibhavArora314.

**Comment by goyalh4164** [comment, marked as answer]
@dhruv389 according to me yesterday because we have merged your PR once and you will not going to get the points if we had closed your PR instead
